user-guide/Advanced_Functionality/Databases/Amazon_Keyspaces_Service.md

```diff
@@ -47,6 +47,38 @@ For Amazon Keyspaces, the Starfield certificate must be present on the local Win
 
    ![Add Certificate](~/user-guide/images/aks_add_certificate.png)
 
+## Generating credentials for Amazon Keyspaces
+
+In the Identity and Access Management (IAM) Service, you can generate new credentials that are used to access the Amazon Keyspaces service. Those credentials are then linked to the account that created them.
+
+In order to do this, the right permissions are required.
+
+All of this takes place in the IAM service.
+
+1. Go to *IAM* > *Users*.
+
+   ![IAM](~/user-guide/images/Amazon_Keyspaces_IAM.png)
+
+1. Search for the account that should have access.
+
+   ![SearchUser](~/user-guide/images/Amazon_Keyspaces_SearchUser.png)
+
+1. Verify that this account has the permissions to generate Amazon Keyspaces credentials.
```

Comment:
What if the account doesn't have them? The way the procedure is written now, you just check if the permissions are there, and then just continue regardless of the outcome of the check. Something also seems to be missing between this step and the next: shouldn't the user log in with that account? If you assume that they are already logged in with this, that should be made clear from the start.

Reply:
Added instructions on how to add the permission.

Also added a first step to log in on AWS and go to the IAM service. Clicking on this link automatically prompts a login screen.